README.md

```diff
@@ -35,7 +35,28 @@ At this stage your code will be better documented and more readable. Perhaps you
 Set the verbosity level of output from MIRAI by setting the environment variable `MIRAI_LOG` to one of
 `info`, `warn`, `debug`, or `trace`.
 
-To go beyond this, super lint + better documentation phase, you'll need to be aware of a lot more things.
+You can also use the environment variable `MIRAI_FLAGS` to provide options to MIRAI. The value is a string which
+can contain any of the following flags:
+
+- `--test_only`: instructs MIRAI to analyze only test methods in your crate. You must also provide the `--tests`
+  option to the `cargo build` command to include those tests actually into your build.
+- `--flaky`: instructs MIRAI to disable measurements for omitting false positives. The output may be a lot
```

Comment:
flaky is generally a pejorative term and we should avoid it. How about --verbose?

Reply:
Flaky is a standard notion in test engineering over the industry. It was also introduced right away in the test talk in bootcamp.

Reply:
I'm also not happy with --flaky, but --verbose seems to be even more off, because it has a particular meaning in CLIs which is not exactly what we are doing here. I was also thinking of --aggressive, --imprecise,  --loose, --unfettered, --unfiltered. Any preferences or additional ideas?

Reply:
Let's call it "--sound" then. It may not be completely sound now, but we can make that more true as time goes on.

"disable measurements" -> "disable measures".

"may include false negatives" -> "may include true positives"